aidants_connect_web/templates/public_website/guide_utilisation.html

```diff
@@ -339,8 +339,7 @@ <h3>
                 <div class="fr-grid-row fr-grid-row--center">
                     <div class="fr-grid-row fr-col fr-grid-row--center">
                         <ul>
-                            <li><a href="{% static 'guides_aidants_connect/Franceconnect_3.pdf' %}" target="_blank">Télécharger en couleur</a></li>
-                            <li><a href="{% static 'guides_aidants_connect/Franceconnect_3_N_B.pdf' %}" target="_blank">Télécharger pour impression en noir et blanc</a></li>
+                            <li><a href="{% static 'guides_aidants_connect/Franceconnect_4.pdf' %}" target="_blank">Télécharger en couleur</a></li>
```

Comment:
```suggestion
                            <li><a href="{% static 'guides_aidants_connect/Aidants Connect _ Quel-fournisseur-identité-choisir.pdf' %}" target="_blank">Télécharger en couleur</a></li>
```
Le PDF existe déjà sous le nom `Aidants Connect _ Quel-fournisseur-identité-choisir.pdf`. Il a été rajouté dans #1094.

Reply:
oui je sais je l'ai renommé, mais comme on a gardé les historique FC_1, FC_3, j'ai préféré rester homogène et mettre un FC_4, mais allez, soyons fou, je le duplique pas